File: ioi/2018/practice/xylophone/xylophone/pas/xylophone.pas
unit xylophone;

interface

procedure detect_pitch(N: longint);

implementation

uses xylophone_lib;


procedure detect_pitch(N: longint);
var
  x: longint;
  i: longint;
begin
  x := ask(0, N - 1);
  for i := 0 to N - 1 do begin
    answer(i, i);
  end;
end;


end.

File: ioi/2018/practice/xylophone/xylophone/java/xylophone.java
public class xylophone {
  public void detect_pitch(int N) {
    int x = xylophone_lib.ask(0, N - 1);
    for (int i = 0; i < N; ++i) {
      xylophone_lib.answer(i, i);
    }
  }
}
